FirebaseAuth.podspec

```diff
@@ -47,10 +48,12 @@ supports email and password accounts, as well as several 3rd party authenticatio
   s.ios.framework = 'SafariServices'
   s.dependency 'FirebaseCore', '~> 6.8'
   s.dependency 'GoogleUtilities/AppDelegateSwizzler', '~> 6.5'
-  s.dependency 'GoogleUtilities/Environment', '~> 6.5'
+  s.dependency 'GoogleUtilities/Environment', '~> 6.6'
```

Comment:
I wonder is there an issue with bumping GoogleUtilities version in this PR as well? Since we use local specs in the tests it should not break anything.

Would you like to update GoogleUtilities changelog in the PR as well?

Reply:
Yep GoogleUtilities should bump to 6.7 here and in its podspec for the new API